test-cases/longevity/longevity-gce-custom-d1-workload2-hybrid-raid.yaml

```diff
@@ -3,7 +3,7 @@ n_monitor_nodes: 1
 n_db_nodes: '5 5'
 n_loaders: '1 1'
 simulated_regions: 2
-
+rack_aware_loader: true
```

Comment:
This scenario doesn't use C-S. It uses latte. And your new option gets used only for C-S.
So, this change is redundant.
Moreover, this scenario is multi-dc, not multi-rack.